ui-templates/custom-types/camera_data.json

```diff
@@ -9,37 +9,37 @@
                 "description": "position of the camera",
                 "required": false,
                 "default": {
-                    "0": 0.0,
-                    "1": 0.0,
-                    "2": 120.0
+                    "x": 0.0,
+                    "y": 0.0,
+                    "z": 120.0
```

Comment:
is it possible to get these defaults from the data?

Reply:
Honestly, we don't inherently need to define the defaults here, since the autoconversion code takes care of that. Plus, I'm not sure if anything happens with these default values currently, since they aren't pre-populated in the smoldyn form and they don't appear in the json object when the form is submitted... Maybe we should just strip this out?

Also from some quick research, I think bringing the defaults in may be possible, but not simple. I think we'd need to find some sort of json library to use that supports that, regular old json doesn't do variables.